src/hooks/useSignInDialog/Steps/AccountStep.tsx

```diff
@@ -0,0 +1,148 @@
+import accountCreation from '@/assets/account-creation.png'
+import { useActiveUser, useJoystream } from '@/hooks'
+import { Placeholder, Spinner, Text } from '@/shared/components'
+import { transitions } from '@/shared/theme'
+import React, { FormEvent, useState } from 'react'
+import { CSSTransition, SwitchTransition } from 'react-transition-group'
+import {
+  AccountStepImg,
+  AccountsWrapper,
+  AccountWrapper,
+  AccountInfo,
+  IconWrapper,
+  OrderedSteps,
+  OrderedStep,
+  IconGroup,
+  AccountAddress,
+  StyledRadioButton,
+  StyledButton,
+} from './AccountStep.style'
+import polkadotIcon from '@/assets/polkadot.png'
+import joystreamIcon from '@/assets/logo.png'
+import { StepFooter, BottomBarIcon, StepSubTitle, StepTitle, StepWrapper } from './Steps.style'
+import { useNavigate } from 'react-router'
+import { SvgGlyphNewChannel, SvgOutlineConnect } from '@/shared/icons'
+
+const AccountStep: React.FC = () => {
+  const navigate = useNavigate()
+  const { setActiveUser } = useActiveUser()
+  const { accounts } = useJoystream()
+  const [imageLoaded, setImageLoaded] = useState(false)
+  const [selectedAccountAddress, setSelectedAccountAddress] = useState<undefined | string>()
+
+  const handleSubmitSelectedAccount = async (e: FormEvent) => {
+    e.preventDefault()
+    if (selectedAccountAddress) {
+      await setActiveUser({
+        accountId: selectedAccountAddress,
+        memberId: null,
+        channelId: null,
+      })
+      navigate('?step=2')
+    }
+  }
+
+  const handleSelect: (e: React.MouseEvent<HTMLElement>) => void = (e) => {
+    const element = e.currentTarget as HTMLInputElement
+    setSelectedAccountAddress(element.value)
+  }
+
+  return (
+    <SwitchTransition>
+      <CSSTransition
+        key={!accounts?.length ? 'no-accounts' : 'accounts'}
+        classNames={transitions.names.fadeAndSlide}
+        timeout={parseInt(transitions.timings.routing)}
+      >
+        {!accounts?.length ? (
+          <StepWrapper centered withBottomBar>
+            {!imageLoaded && <Placeholder height="180px" width="320px" />}
+            <CSSTransition in={imageLoaded} classNames="fade" timeout={100}>
+              <AccountStepImg src={accountCreation} onLoad={() => setImageLoaded(true)} />
+            </CSSTransition>
```

Comment:
Why do we need to handle image loading here?

Reply:
I was experiencing a small layout shift here, but I will get rid of that. I'll just set a fixed height to the image.